### PR TITLE
Added Backoffice Settings article

### PR DESCRIPTION
**Pull request by eshanrnh**
- [x] Add v10 article after review.

### CODE REVIEW

File: 11/umbraco-cms/fundamentals/backoffice/settings.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+- Internals - Lets you trigger a NuCache snapshots collection.
+
+</details>
+
+<details>
+
+<summary>Models Builder</summary>
+
+Models builder is a tool that can generate a complete set of strongly-typed published content models for Umbraco. Models are available in both controllers and views.
```
Comment (sofietoft) [inline 11/umbraco-cms/fundamentals/backoffice/settings.md:47]: Would be great with a bit more detail about the dashboard itself. I believe it's possible to rebuild/reload/generate models when it is enabled!

File: 11/umbraco-cms/fundamentals/backoffice/settings.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+</details>
+
+<details>
+
+<summary>Health Check</summary>
+
+Health Checks are used to determine the status of your Umbraco project. It is a handy list of checks to see if your Umbraco installation is configured according to best practices. It's possible to add your custom-built health checks.
+
+For more information about Health Checks, see the [Health Check](health-check/README.md) article.
```
Comment (sofietoft) [inline 11/umbraco-cms/fundamentals/backoffice/settings.md:59]: Added an "s" to indicate there are many articles. Feel free to remove if it doesn't make sense 😅 

```suggestion
For more information about Health Checks, see the [Health Check](health-check/README.md) articles.
```

File: 11/umbraco-cms/fundamentals/backoffice/settings.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+For more information about MiniProfiler, see the [MiniProfiler](../code/debugging/README.md#miniprofiler) section in the [Debugging](../code/debugging/README.md) article.
+
+</details>
+
+<details>
+
+<summary>Telemetry Data</summary>
+
+The Telemetry Data dashboard is a consent screen that is used for collecting system and usage information from your installation. Here, you can see what type of data is being collected and even adjust the level of reporting. Currently, there are three levels available: **Minimal**, **Basic**, and **Detailed**. **Basic** is the default option where the data sent is an anonymized site ID, Umbraco version, and packages installed. You can see the specific data being sent on each of the levels directly in the Dashboard.
```
Comment (sofietoft) [inline 11/umbraco-cms/fundamentals/backoffice/settings.md:79]: As of the latest version of CMS (in RC currently), the default option is "Detailed".

```suggestion
The Telemetry Data dashboard is a consent screen that is used for collecting system and usage information from your installation. Here, you can see what type of data is being collected and even adjust the level of reporting. Currently, there are three levels available: **Minimal**, **Basic**, and **Detailed**. **Basic** is the default option where the data sent is an anonymized site ID, Umbraco version, and packages installed. You can see the specific data being sent on each of the levels directly in the Dashboard.
```

File: 11/umbraco-cms/fundamentals/backoffice/settings.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+For more information about Package Telemetry, see the [Package Telemetry](https://umbraco.com/blog/umbraco-92-release/) section in the Umbraco 9.2 Release Blog Post.
```
Comment (sofietoft) [inline 11/umbraco-cms/fundamentals/backoffice/settings.md:81]: I think this makes it sounds like it's only package telemetry that's being gathered 😅  Would be good with some info before this, about how the package telemetry is also gathered as part of this.

File: 11/umbraco-cms/fundamentals/backoffice/settings.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
+---
+description: A guide displaying the options available in the Settings section in Umbraco Backoffice.
+---
+
+# Backoffice Settings Section
```
Comment (sofietoft) [inline 11/umbraco-cms/fundamentals/backoffice/settings.md:5]: Remember that this title needs to be the same as the title you give it in the navigation.

Also, as this article is only about Dashboards in the Settings section, I think the name/title should reflect that!

**Review by sofietoft — COMMENTED** [review]
Super article! 💪 
I've added some comments.

Also, would it make sense to link to this article from the "Sections" article, also in the Backoffice part of the Fundamentals?

File: 11/umbraco-cms/fundamentals/backoffice/settings.md (hunk 5 above)
Comment (eshanrnh) [reply 11/umbraco-cms/fundamentals/backoffice/settings.md:5]: I'll change the article name to "Dashboards in the Settings section"

Comment (sofietoft) [reply 11/umbraco-cms/fundamentals/backoffice/settings.md:5]: Of if that's too long, perhaps "Settings Dashboards"? 🤔

File: 11/umbraco-cms/fundamentals/backoffice/settings.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+# Settings Dashboards
+
+The **Settings** section of the Umbraco backoffice has its own set of default dashboards. In this article, you can get an overview of each dashboard avaiable in the **Settings** section:
+
+<details>
+
+<summary>Welcome</summary>
+
+The Welcome dashboard is the first dashboard in the Settings section. Like all dashboards, it is displayed next to the Settings tree and consists of a view that can be customized. The Welcome dashboard consists of links to different resources or items in the Settings section.
```
Comment (sofietoft) [inline 11/umbraco-cms/fundamentals/backoffice/settings.md:13]: It's not linking the anything in the Settings section 🙈 

```suggestion
The Welcome dashboard is the first dashboard in the Settings section. Like all dashboards, it is displayed next to the Settings tree and consists of a view that can be customized. The Welcome dashboard consists of links to different resources that can be used when developing your Umbraco website.
```

**Review by sofietoft — COMMENTED** [review]
Thanks for applying the suggestions 💪 
I've left one more comment, and also - should the file name be updated, to match the title as well?

**Comment by eshanrnh** [comment]
Done 💪 Changed the file name as well.

**Pull request merged**